docs/_data/menu-sql.yaml

```diff
@@ -72,12 +72,164 @@
       url: sql-migration-guide-hive-compatibility.html
     - text: SQL Reserved/Non-Reserved Keywords
       url: sql-reserved-and-non-reserved-keywords.html
-- text: Reference
-  url: sql-reference.html
+
+- text: SQL Reference
+  url: sql-ref.html
   subitems:
     - text: Data Types
-      url: sql-reference.html#data-types
+      url: sql-ref-datatypes.html
     - text: NaN Semantics
-      url: sql-reference.html#nan-semantics
-    - text: Arithmetic operations
-      url: sql-reference.html#arithmetic-operations
+      url: sql-ref-nan-semantics.html
+    - text: SQL Syntax
+      url: sql-ref-syntax.html
+      subitems:
+        - text: Data Definition Statements
+          url: sql-ref-syntax-ddl.html
+          subitems:
+            - text: ALTER DATABASE
+              url: sql-ref-syntax-ddl-alter-database.html
```

Comment:
Could you create sub tasks for each page?

Reply:
@gatorsmile Sure.. I will create the sub tasks tomorrow.